pyo3-ffi/src/object.rs

```diff
@@ -62,11 +76,21 @@ pub unsafe fn Py_Is(x: *mut PyObject, y: *mut PyObject) -> c_int {
 // skipped _Py_REFCNT: defined in Py_REFCNT
 
 #[inline]
+#[cfg(not(Py_NOGIL))]
 pub unsafe fn Py_REFCNT(ob: *mut PyObject) -> Py_ssize_t {
     assert!(!ob.is_null());
     (*ob).ob_refcnt
 }
 
+#[inline]
+#[cfg(Py_NOGIL)]
+pub unsafe fn Py_REFCNT(ob: *mut PyObject) -> Py_ssize_t {
+    if ob.is_null() {
+        panic!();
```

Comment:
Why this instead of functionally equivalent but more readable `assert!` like above?

Reply:
Fixed. This was an artifact from rebasing the 0.15.2 changes where the above function used an `if` statement.